### ISSUE TITLE
Wireless Controller not found in Driver installer. It is connected via bluetooth.

### ISSUE THREAD

**Issue by Ichigo07** [nefarius/ScpToolkit#173]
[ScpToolkit_Log-Package_Ichigo_1453294942.zip](https://github.com/nefarius/ScpToolkit/files/97456/ScpToolkit_Log-Package_Ichigo_1453294942.zip)

**Comment by Jan200101** [comment]
Wireless controllers wont be picked up my the installer. You have to
connect it over cable

Sentry

**Comment by Ichigo07** [comment]
I was playing with this exact setup with the old ScpServer which was working until like 2 month ago.

When i try to connect it via cable it works with the old scpServer. 

Somehow there are always 2 Xbox Controller and 1 wireless Controller showed in the Controller menu with the old scpServer.

I recognized that the controller seems to work as second controller with the old scp server.

It is an Original Dualshock 4.

**Comment by Jan200101** [comment]
Do you have any virtual controllers (e.g. Vjoy) installed?

**Comment by Ichigo07** [comment]
[ScpToolkit_Log-Package_Ichigo_1453295572.zip](https://github.com/nefarius/ScpToolkit/files/97467/ScpToolkit_Log-Package_Ichigo_1453295572.zip)

i installed it now via cable and its flashing blue now. But its not working ingame.

i havent installed Vjoy, i only had Scpserver (the old one) which worked 3 days ago for some hours, then i had ds4windows which i deinstalled (even the drivers) and i got this one.

**Comment by Ichigo07** [comment]
![3](https://cloud.githubusercontent.com/assets/16100294/12449994/4da6a714-bf81-11e5-8fdf-8c1bb25ca9b8.jpg)
![2](https://cloud.githubusercontent.com/assets/16100294/12449995/4f72c244-bf81-11e5-8bf5-3fe7cce4c936.jpg)
![unbenannt](https://cloud.githubusercontent.com/assets/16100294/12449998/52b7604a-bf81-11e5-87cf-1240f1650363.jpg)

**Comment by Ichigo07** [comment]
this was a wired installation, if its only connected via bluetooth there is no wireless controller listed

**Comment by Ichigo07** [comment]
Input tester is working tho...

**Comment by Jan200101** [comment]
okay can you make a log with the controller connected over cable?

**Comment by Ichigo07** [comment]
[ScpToolkit_Log-Package_Ichigo_1453296731.zip](https://github.com/nefarius/ScpToolkit/files/97499/ScpToolkit_Log-Package_Ichigo_1453296731.zip)

I really appriciate your help, i was searching for the problem 2 days now and i wasnt able to find any clue why its acting like that. because it was working with the old scpserver sometimes, even wireless.

**Comment by Jan200101** [comment]
btw is the service on?

**Comment by Ichigo07** [comment]
yes its active atm

**Comment by Jan200101** [comment]
Okay try disableing the one that is not there

**Comment by Jan200101** [comment]
its called Xbox 360 Controller for Windows

**Comment by Ichigo07** [comment]
![4](https://cloud.githubusercontent.com/assets/16100294/12450431/021ca07a-bf84-11e5-8a25-212a45116ee7.jpg)

**Comment by Ichigo07** [comment]
There is only one atm

**Comment by Jan200101** [comment]
okay win10 is very confusing :I
i am going to replicate it and find out how to fix

**Comment by Jan200101** [comment]
The Usb file in you log says that there are 2

**Comment by Jan200101** [comment]
Her
1118;654;Controller (XBOX 360 For Windows);USB\VID_045E&PID_028E&IG_03\2&9C6DEB5&B&03;USB\VID_045E&PID_028E&IG_03;HidUsb;Unknown

and here 

1118;654;Xbox 360 Controller for Windows;USB\VID_045E&PID_028E\0000001;USB\VID_045E&PID_028E&REV_0114;xusb22;Unknown

**Comment by Ichigo07** [comment]
On the old scpserver it was acting like that, on some games the controller worked wireless but it was always connected as second player, so i couldnt play. But the game recognized the controller.

**Comment by Ichigo07** [comment]
let me try to restart then i will redo the debug lets see. brb

**Comment by Ichigo07** [comment]
[ScpToolkit_Log-Package_Ichigo_1453298164.zip](https://github.com/nefarius/ScpToolkit/files/97517/ScpToolkit_Log-Package_Ichigo_1453298164.zip)

**Comment by Ichigo07** [comment]
okay now it seems to work wired. so how will i be able to get it wireless working?

**Comment by Ichigo07** [comment]
I tried to connect via bluetooth now by just pressing the playstation button and it said dualshock 4 controller found but the controller instantly shuts off.

**Comment by Uhsheen** [comment]
I just picked up a Bluetooth adapter today and I am having the same issue.

**Comment by Ichigo07** [comment]
I havent found a solution for the Bluetooth way yet.

**Comment by war1ox** [comment]
I have the same connection problem, I install everything correctly and even restart. The Bluetooth is connected but wont work in any games. The version that was released within the last 2 updates worked flawlessly, and now it seems like it doesn't work at all and I have the real dualshock 3.

**Comment by nefarius** [comment]
I just love it when older releases work better than newer ones -_- I will investigate...

**Comment by KinanEldari** [comment]
"I tried to connect via bluetooth now by just pressing the playstation button and it said dualshock 4 controller found but the controller instantly shuts off. "

I fixed this by first doing the clean uninstall utility (if your Pc can handle it), then connecting the DS4 through Windows' regular bluetooth utility first and making sure it's paired there. After that, I ran the installation and initialization setup again and my controller once again worked. Mind, this is with the 1.7Beta drivers, but it's worth a shot, right?
